tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_pip.sh

```diff
@@ -91,11 +98,13 @@ export TF_TEST_TARGETS="${DEFAULT_BAZEL_TARGETS} \
     -//tensorflow/python/kernel_tests/nn_ops:conv_ops_test \
     -//tensorflow/python/kernel_tests/nn_ops:conv2d_backprop_filter_grad_test \
     -//tensorflow/python/kernel_tests/nn_ops:conv3d_backprop_filter_v2_grad_test \
+    -//tensorflow/python/kernel_tests/nn_ops:pooling_ops_3d_test_cpu \
```

Comment:
This test is for CPU though?

Reply:
Ah, yes, that was a newly failing test. So it really should have been added in a separate change rather than included in this one. However it has since been fixed and is no longer needed here so I am about to update this PR to rebase to current HEAD to resolve the merge conflicts, which will remove this added test.